src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCounter = 1;
+
+  const getSum = (num) => {
+    return (sum) => {
+      if (callCounter >= 4 && callCounter % 2 === 0) {
+        callCounter++;
+
+        return 'Bzzz... Error!';
+      }
+      callCounter++;
```

Comment:
I think it would be more logical to avoid duplication by increasing `callcounter` at the very start of every function call despite any condition. 
```suggestion
  let callCounter = 0;

  const getSum = (num) => {
    return (sum) => {
      callCounter++;

      if (callCounter >= 4 && callCounter % 2 === 0) {
        return 'Bzzz... Error!';
      }
```